infra.bs

```diff
@@ -628,38 +628,40 @@ them effectively as-is.
 of the <a>code units</a> 0xD83D, 0xDCA9, and 0xD800, when interpreted as containing
 <a>code points</a>, would consist of the <a>code points</a> U+1F4A9 and U+D800.
 
-<p>A <dfn export>scalar value string</dfn> is a sequence of <a>scalar values</a>.
+<p>A <a>string</a>'s <dfn export for=string>code unit length</dfn> is the number of
+<a>code units</a> it contains. For now, this can also be referred to as a <a>JavaScript string</a>'s
+<dfn export for="JavaScript string">length</dfn>.
+
+<p>A <a>string</a>'s <dfn export for=string>code point length</dfn> is the number of
+<a>code points</a> it contains. For now, this can also be referred to as a <a>string</a>'s
+<dfn export for=string>length</dfn>.
+
```

Comment:
Perhaps we should add an XXX box with an issue reference to explain the "for now"?